kong/dao/schemas_validation.lua

```diff
@@ -83,6 +83,9 @@ function _M.validate_entity(tbl, schema, options)
 
       if not partial_update then
         for column, v in pairs(schema.fields) do
+          if t[column] == ngx.null then
+            t[column] = nil
+          end
```

Comment:
Note to self: so if we do a full update (`PUT`), and we set a field with a "default" attribute to `NULL`, we actually pull a new value from the configured "default" attribute (text or function). The behavior is fine (I think?), as most fields with a "default" values are so because they are not supposed to be empty. But it is still a somewhat unexpected behavior I think. Nothing else to add, just dropping a note here...